README.md

```diff
@@ -22,12 +22,22 @@ sudo make install
 ```bash
 mkdir myproject && cd myproject
 
+For a GitHub repo:
+
 echo '{
   "deps": [
     { "type": "github", "repo": "jemc/pony-inspect" }
   ]
 }' > bundle.json
 
+For a local git repo:
+
+echo '{
+  "deps": [
+    { "type": "local-git", "package-name": "inspect", "local-path": "../pony-inspect" }
+  ]
+}' > bundle.json
+
```

Comment:
It probably makes sense to also add an example of `local`, right?

Also, the `local-git` example should probably include a `tag`, since that seems to be the only reason why you would want to use `local-git` instead of just `local`, right?